Wrapping/Generators/Python/itk/support/base.py

```diff
@@ -321,19 +347,17 @@ def load_factories(factory_name: str) -> None:
     # Only load factories once
     if factory_name in _factories_loaded:
         return
-    import importlib
+
     import itk
     for module_name, data in itk_base_global_module_data.items():
         for name, factory_class_prefix in data.get_module_factories()[:2]:
             if name == factory_name:
-                module = importlib.import_module(f"itk.{module_name}Python")
-                factory = getattr(module, f"itk{factory_class_prefix}{name}Factory")
-                # Cache
-                cur = itk.auto_pipeline.current
```

Comment:
This was added to prevent circular dependencies. There may be other code paths like this to address.

Reply:
I was able to achieve the same desired functionality with `itk_load_swig_module` by moving where `__templated_loaded` is defined. Note that the `import_module` call here resulted in the `PyCapsule` error on my machine that I described earlier, though I'm still not sure of the exact reason. However, `itk_load_swig_module` can offer more benefits to early symbol loading.